server/src/main/java/org/elasticsearch/common/regex/Regex.java

```diff
@@ -75,21 +75,31 @@ public static Automaton simpleMatchToAutomaton(String... patterns) {
         return Operations.union(automata);
     }
 
+    // Case sensitive version of simple match
```

Comment:
I'd copy the javadoc too, just so that when I hover over the method call I can see it.